scriptworker/test/test_cot_verify.py

```diff
@@ -1376,10 +1376,6 @@ def test_verify_link_in_task_graph_exception(chain, decision_link, build_link, i
         },
         'now': '2018-01-01T12:00:00.000Z',
         'ownTaskId': 'decision_task_id',
-        'repository': {
```

Comment:
:+1: Stripping `jsone_context['repository']` for Github events makes sense to me!